src/test/java/org/assertj/core/extention/SoftlyExtensionExecutionTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.extention;
+
+import org.assertj.core.annotations.Softly;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.SoftAssertions;
+import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
+import org.assertj.core.extention.SoftlyExtension;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.assertj.core.util.AssertionsUtil.assertThatAssertionErrorIsThrownBy;
+
+@DisplayName("SoftlyExtension runs")
+@ExtendWith({SoftlyExtension.class, MockitoExtension.class})
+class SoftlyExtensionExecutionTest {
+
+  @Softly(assertAfter = true)
+  @InjectMocks
```

Comment:
Why do you add this mockito annotation ?

Reply:
This is a mistake that I will rectify. I was trying to write a test that would assert that #assertAll () would be called after the test run. I was thinking of somehow using Mockito to get the test instance then verify that the method was called. But it was not possible. Is it possible to test this?